File: HA2_1.cpp
//Accept one number from user and print that number of * on screen

#include<iostream>
using namespace std;

void Display(int iNo)
{
    int iCnt = 0;

    for(iCnt = 1; iCnt<= iNo; iCnt++)
    {
        cout<<"*"<<"\n";
    }
}

int main()
{
    int iValue = 0;
    cout<<"Enter number"<<"\n";
    cin>>iValue;

    Display(iValue);

    return 0;
}

File: HA2_1.py
#Accept one number from user and print that number of * on screen

def Display(iNo):
    for iCnt in range(0, iNo, 1):
        print("*")

def main():
    iValue = 0

    print("Enter a number")
    iValue = int(input())

    Display(iValue)

if __name__=="__main__":
    main()